install.sh

```diff
@@ -83,65 +71,74 @@ done
 
 # Conda does not like ~
 conda_prefix=${conda_prefix//\~/$HOME}
-mcprefix=${mcprefix//\~/$HOME}
-export EMIRGE_MIRGECOM_BRANCH=$mcbranch
+mcprefix=$SCRIPT_DIR
 export MY_CONDA_DIR=$conda_prefix
+export MY_CONDA_ENV=$env_name
 
 ./install-conda.sh
 
-export PATH=$MY_CONDA_DIR/bin:$PATH
-
-echo "==== Create $env_name conda environment"
-
 # Make sure we get the just installed conda.
 # See https://github.com/conda/conda/issues/10133 for details.
 #shellcheck disable=SC1090
 source "$MY_CONDA_DIR"/bin/activate
 
-conda create --name "$env_name" --yes
+# Check if environment already exists
+set +o errexit
+has_env=$(conda info --envs | awk '{print $1}' | grep "$env_name")
+set -o errexit
+
+if [[ $has_env != "$env_name" ]]; then
+  echo "==== Creating $env_name conda environment"
+  conda create --name "$env_name" --yes
+else
+  echo "!!!! Using exisiting $env_name environment"
+fi
 
 #shellcheck disable=SC1090
 source "$MY_CONDA_DIR"/bin/activate "$env_name"
 
-mkdir -p "$mcprefix"
-mcsrc="$mcprefix/mirgecom"
+echo "==== Installing conda packages"
 
-./fetch-mirgecom.sh "$mcbranch" "$mcprefix"
 ./install-conda-dependencies.sh
-if [[ ! -z "$conda_pkg_file" ]]; then
-    ./install-conda-dependencies.sh "$conda_pkg_file"
-fi
-./install-pip-dependencies.sh "$mcsrc/requirements.txt" "$mcprefix"
-if [[ ! -z "$pip_pkg_file" ]]; then
-    ./install-pip-dependencies.sh "$pip_pkg_file" "$mcprefix"
-fi
-./install-src-package.sh "$mcsrc" "develop"
+[[ -n "$conda_pkg_file" ]] && ./install-conda-dependencies.sh "$conda_pkg_file"
+
+echo "==== Installing pip packages for general development (requirements_dev.txt)"
+
+python -m pip install -r requirements_dev.txt
+
+echo "==== Installing packages for mirgecom (requirements.txt)"
+
+grep -E -v '(islpy|pyopencl)' requirements.txt >> .req
```

Comment:
Given `#egg=` annotations, `pip` should not reinstall these if they've been installed by conda. So this should not be necessary. (Though I don't know how this interacts with `--editable`.)

Reply:
Yeah, --editable seems to cause reinstallation (out of curiosity, do we expect to work on islpy)?

Reply:
> (out of curiosity, do we expect to work on islpy)

Not super likely.